sssom/constants.py

```diff
@@ -0,0 +1,7 @@
+"""Constants."""
+
+import os
+import pathlib
+
+CWD = pathlib.Path(__file__).parent.resolve()
```

Comment:
This is the only line I cant judge. If you are positive this works with pip install, I am happy!

Reply:
yes this does what you expect! however, `CWD` is a very misleading name, since this is not at all related to the notion of `os.getcwd()`. I'd suggest renaming this to `HERE`

```suggestion
HERE = pathlib.Path(__file__).parent.resolve()
```

Reply:
Done